CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 
 ### Official Theme release
 
-Gradio now supports a new theme system, which allows you to customize the look and feel of your app. You can now use the `theme=` kwarg to pass in a prebuilt theme, or customize your own! See guides/06_other-tutorials/theming-guide for more details. By [@aliabid94](https://github.com/aliabid94) in [PR 3494](https://github.com/gradio-app/gradio/pull/3494) 
+Gradio now supports a new theme system, which allows you to customize the look and feel of your app. You can now use the `theme=` kwarg to pass in a prebuilt theme, or customize your own! See guides/06_other-tutorials/theming-guide for more details. By [@aliabid94](https://github.com/aliabid94) in [PR 3470](https://github.com/gradio-app/gradio/pull/3470) and [PR 3497](https://github.com/gradio-app/gradio/pull/3497)
```

Comment:
I think `guides/06_other-tutorials/theming-guide` won't actually link to the guide:

![image](https://user-images.githubusercontent.com/41651716/226061900-1cdccf9a-3218-481b-b37e-21153572d47d.png)

Reply:
I actually hadn't meant for it to be a link, just a reference within the repo where the guide is. I could link to http://gradio.app/theming-guide instead?

Reply:
I think that would make sense!

Reply:
Agree